### PR TITLE
Add children to the props for @types/react@18

### PR DESCRIPTION
**Pull request by ChromeQ**
In relation to React props should not do anything special with children and DefinitelyTyped have updated the react types to v18

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/59802

### CODE REVIEW

**Comment by vdv73rus** [comment]
Also waiting for that! 👍🏼

**Review by vdv73rus — APPROVED** [review]

**Comment by ChromeQ** [comment]
2 approves and still complaining of unable to merge until an approved approver approves...

**Review by waltjones — APPROVED** [review]

**Comment by waltjones** [comment]
Released in v0.11.1.

**Pull request merged**
